inc/parser/handler.php

```diff
@@ -926,21 +974,22 @@ public function rss($match, $state, $pos) {
      * @param int $pos byte position in the original source file
      * @return bool mode handled?
      */
-    public function externallink($match, $state, $pos) {
-        $url   = $match;
+    public function externallink($match, $state, $pos)
+    {
+        $url = $match;
         $title = null;
 
         // add protocol on simple short URLs
-        if(substr($url,0,3) == 'ftp' && (substr($url,0,6) != 'ftp://')){
+        if (substr($url, 0, 3) == 'ftp' && (substr($url, 0, 6) != 'ftp://')) {
             $title = $url;
-            $url   = 'ftp://'.$url;
+            $url = 'ftp://' . $url;
         }
-        if(substr($url,0,3) == 'www' && (substr($url,0,7) != 'http://')){
+        if (substr($url, 0, 3) == 'www' && (substr($url, 0, 7) != 'http://')) {
```

Comment:
If `$url` starts with `'www'` then of course it doesn't start with `'http://'`!  So this could be simplified to:
`if (subset($url, 0, 3) == 'www') {` ?

Or maybe this was meant to work differently and contains a mistake? (Also, what about `'https://'` if so?)